src/Bookstore.Service/Startup.cs

```diff
@@ -36,6 +36,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllers();
             services.AddSwaggerGen(c =>
             {
+                //By default the type name is used for the schema id.
+                //This can generate a conflict if we have multiple entites with the same name but in different modules
+                //so we are using the full type name for the schema id.
+                c.CustomSchemaIds(type => type.FullName);
```

Comment:
This solution resolves mentioned problem but messes up naming of generic classes. 

For Rhetos, a lot of schemas like this are generated `Rhetos.Host.AspNet.RestApi.Utilities.RecordsResult`1[[AspNetDemo.AnotherEntity, RhetosApp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]`

Default Swagger strategy for schema generation is:
[GitHub source](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/539f96b0a1124c77da29b0eefc0ef185dd0c5230/src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/SchemaGeneratorOptions.cs#L44)

Unfortunately, that is private. Maybe roll our own helper similar to this one?

Reply:
Vjerojatno je najbolje preuzeti ovaj kod od swashbucklea, i doraditi da se doda fullname za prvi naveden tip (vjerojatno zadnji tip u petlji).
Nije problem ako ovime promijenimo i metadatu za non-rhetos servise, pošto developer može napisati vlastiti delegat ako želi.
Izdvojiti taj delegato kao helper metodu u RestGenerator-u, koju će ljudi lako pozvati iz svoje konfiguracije.

Reply:
Keeping the current solution, for simplicity. The [official documentation](https://github.com/domaindrivendev/Swashbuckle.AspNetCore#customize-schema-ids) uses FullName as an example, so it shouldn't be an issue.
We could use ToString instead of FullName to remove assembly information from generic types (version, culture, public key token).